resources/docs/Documentation.md

```diff
@@ -62,42 +62,42 @@ RSS Guard is simple (yet powerful) feed reader. It is able to fetch the most kno
     * Inoreader (RSS Guard 3.5.0+),
     * Gmail with e-mail sending (RSS Guard 3.7.1+).
 * core:
-    * support for all feed formats (RSS/RDF/ATOM/JSON),
-    * full support of podcasts (RSS/ATOM/JSON),
-    * import/export of feeds to/from OPML 2.0,
-    * possibility of using custom 3rd-party feed synchronization services,
-    * feed metadata fetching including icons,
-    * simple internal Chromium-based web viewer (or alternative version with simpler and much more lightweight internal viewer),
-    * scriptable [message filtering](#message-filtering),
-    * downloader with own tab and support for up to 6 parallel downloads,
-    * ability to cleanup internal message database with various options,
-    * enhanced feed auto-updating with separate time intervals,
-    * "portable" mode support with clever auto-detection,
-    * feed categorization,
-    * feed authentication (BASIC),
-    * handles tons of messages & feeds,
-    * ability to backup/restore database or settings,
-    * fully-featured recycle bin,
-    * multiple data backend support,
+    * Support for all feed formats (RSS/RDF/ATOM/JSON),
```

Comment:
AP style book recommends lists to be started with capitalized words; I've found it better for texts to be written in a familiar style to avoid distracting from the small things. Some people will pay more attention to these things than to the content of the document itself, sadly.